az-digital/az_quickstart
# Closes #2712: Fix sidebar news heading level accessibility

## Description
Update heading levels to meet accessibility requirements

## Related issues
Closes #2712

## How to test
Probo build example: https://31b9dd0d-01d0-487e-a718-d198d2ff68bb.probo.build/news/could-mans-best-friend-be-mans-best-medicine
1. Open a news article
2. View the sidebar news items
3. Inspect to ensure heading level is H3 tag
4. Ensure all other styling stayed the same

## Types of changes

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
